nbconvert/preprocessors/csshtmlheader.py

```diff
@@ -7,11 +7,19 @@
 import os
 import io
 import hashlib
+import nbconvert.resources
 
 from traitlets import Unicode
 from ipython_genutils.py3compat import str_to_bytes
 from .base import Preprocessor
 
+
+try:
+    from notebook import DEFAULT_STATIC_FILES_PATH
```

Comment:
Because otherwise that's make the notebook a hard dependency for all of nbconvert, while as local import it's only a dependency if this preprocessor is actually called.